### DISCUSSION TITLE
supporting APC Codes: Ambulatory Payment Classifications

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/price-transparency-guide#61] [Q&A] [answered]
In addition to DRG codes some hospitals use APC codes - Ambulatory Payment Classifications.
While they are officially the government's method of paying for facility outpatient services for the Medicare program.  It seems that they are also sometimes used by commercial payers.

Should they be included?

**Comment by tdfow** [comment]
also related to discussion/thread in  #9

**Comment by shaselton-usds** [comment, marked as answer]
@nmayle Please take a look at the answer here: https://github.com/CMSgov/price-transparency-guide/discussions/9
